ext/opcache/ZendAccelerator.c

```diff
@@ -3975,7 +3975,7 @@ static void preload_link(void)
 			if (ce->type == ZEND_INTERNAL_CLASS) {
```

Comment:
Might make more sense to `|| !(ce->ce_flags & ZEND_ACC_LINKED)` here since we already have an early return branch?

Reply:
Do we want to early return from this loop, though? There may still be valid class entries before/after this one

Reply:
Hmm, depends on what would be added, but I guess you're right in that keeping this in the case-specific if statement forces you to think about it. Let's keep it as is then.

Reply:
Oh, this is a break and not continue, sorry I completely missed that.